website/docs/docs/contributing/oss-expectations.md

```diff
@@ -0,0 +1,104 @@
+
+If, over the course of a discussion, we do manage to reach consensus on a way forward, we’ll open a new issue that references the discussion for context. That issue will connect desired outcomes to specific implementation details, as well as perceived limitations and open questions. It will serve as a formal proposal and request for comment.
+
+## Issues
+
+An issue could be a bug you’ve identified while using the product or reading the documentation. It could also be a specific idea you’ve had for how it could be better.
+
+### Best practices for issues
+
+- Issues are **not** for support / troubleshooting / debugging help. Please open a discussion instead (so other future users can search for it), or post in the dbt Slack Community for help understanding the problem you’re running into.
+- Always search existing issues first, to see if someone else had the same idea / found the same bug you did.
+- Many repositories offer templates for creating issues, such as when reporting a bug or requesting a new feature. If available, please select the relevant template and fill it out to the best of your ability. This will help other people understand your issue and respond.
+
+### You’ve found an existing issue that interests you. What should you do?
+
+Comment on it! Explain that you’ve run into the same bug, or had a similar idea for a new feature. If the issue includes a detailed proposal for a change, say which parts of the proposal you find most compelling, and which parts give you pause.
+
+### You’ve opened a new issue. What can you expect to happen?
+
+In our most critical repositories (such as `dbt-core`), **our goal is to respond to most new issues within 2 standard work days.** This response will include the context, feedback, and pointers that we can offer as maintainers. Depending on the nature of your issue, it might be well suited to an external contribution, from you or another community member.
+
+**What does “triage” mean?** In some repositories, we use a `triage` label to keep track of issues that need a response from a maintainer. In the `dbt-core` repo, we also tag issues with a “team” that will be providing the response.
+
+**What if I’m opening an issue in a smaller repository?** We’ve open sourced a number of software projects over the years; not all of them have the same activity or maintenance guarantees as `dbt-core`. Check to see if other recent issues have responses, or when the last commit was added to the `main` branch.
+
+**If my issue is lingering...** Sorry for the delay! If your issue still has the `triage` label attached, we’re still aware of it as something that needs a response. If it’s been more than two weeks without a response, we’d encourage you to comment, reminding us that you’re awaiting.
```

Comment:
Should they @ someone? I know there are some small repos that don't have someone subscribed to all events, unlike the core/utils repos. 

Maybe we want a group (make public the package maintainers group?) or we make a looker/hex dash that pulls through all activities on quiet repos?

Reply:
Both of those are solid ideas. I do get many of these GitHub notifications, they're just buried in a sea of... all these GitHub notifications

I've done some dbt/looker/hex-y work in this vein for the Core team's repos, and I think that's a good thread to pull on. The only downside right now is fairly high latency (once daily syncs from GitHub). Doesn't matter as much in repos where our goal is less around guaranteed timely response, and more around guaranteeing these don't fall off the radar

Reply:
> they're just buried in a sea of... all these GitHub notifications

oof. 💯  😣



> Doesn't matter as much in repos where our goal is less around guaranteed timely response, and more around guaranteeing these don't fall off the radar

Yeah that's what I was thinking. the only other option is to Operationalise Our Analytics™️ and reverse ETL them into Slack or something, but I don't know that that's any better.

Reply:
yeah I'd like some sort of dashboard for this stuff. But if an @ that has a few people behind it is acceptable that also works if it's documented here.

Reply:
Shall we set up `@dbt-labs/oss-maintainers`?

Reply:
Just created @dbt-labs/oss-maintainers. Open to who should be in this group, and what the tagline should be